app/lib/actions/katello/repository/clone_contents.rb

```diff
@@ -6,22 +6,27 @@ def plan(source_repositories, new_repository, options)
           filters = options.fetch(:filters, nil)
           rpm_filenames = options.fetch(:rpm_filenames, nil)
           generate_metadata = options.fetch(:generate_metadata, true)
-          index_content = options.fetch(:index_content, true)
           purge_empty_contents = options.fetch(:purge_empty_contents, false)
           copy_contents = options.fetch(:copy_contents, true)
           solve_dependencies = options.fetch(:solve_dependencies, false)
 
           sequence do
             if copy_contents
               source_repositories.each do |repository|
-                plan_action(Pulp::Repository::CopyAllUnits, repository, new_repository,
+                plan_action(PulpSelector,
+                            [Pulp3::Orchestration::Repository::CopyAllUnits, Pulp::Repository::CopyAllUnits],
+                            repository,
+                            SmartProxy.pulp_master,
+                            new_repository,
                             filters: filters, rpm_filenames: rpm_filenames, solve_dependencies: solve_dependencies)
```

Comment:
This one here seems to not be updated to the new PulpSelctor..

Reply:
updated!